lib/wx/api_gen/wxapi.conf

```diff
@@ -2260,7 +2260,7 @@
 
 {class, wxTaskBarIcon, wxEvtHandler, [],
  [{wxTaskBarIcon, [{where, taylormade}]},'~wxTaskBarIcon',
```

Comment:
@leoliu This one? Perhaps it supposed to be `tailoredmade` or `tailor-made`? Not sure about the historical context.

Reply:
Yes that one. I also thought it was supposed to mean `tailor-made` or similar.